sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -84,27 +84,29 @@ public class ServiceBusSender : IAsyncDisposable
         ///   Initializes a new instance of the <see cref="ServiceBusSender"/> class.
         /// </summary>
         /// <param name="entityPath">The entity path to send the message to.</param>
-        /// <param name="viaEntityPath">The entity path to route the message through. Useful when using transactions.</param>
+        /// <param name="options">The set of <see cref="ServiceBusClientOptions"/> to use for configuring
+        /// this <see cref="ServiceBusClient"/>.</param>
```

Comment:
nit: `ServiceBusClientOptions` -> `ServiceBusSenderOptions` and `ServiceBusClient` -> `ServiceBusSender`